graph/telemetry/istio/appender/appender.go

```diff
@@ -22,6 +22,8 @@ func ParseAppenders(o graph.TelemetryOptions) []graph.Appender {
 	if !o.Appenders.All {
 		for _, appenderName := range o.Appenders.AppenderNames {
 			switch appenderName {
+			case HealthConfigAppenderName:
+				requestedAppenders[HealthConfigAppenderName] = true
```

Comment:
Name change is good. Again, maintain alphabetical ordering where it is being used, it helps in code readability.